### PR TITLE
update manifest

### PR DESCRIPTION
**Pull request by Arhell**

### CODE REVIEW

File: templates/cakephp/@layout.latte [hunk 1]
```diff
@@ -24,7 +24,7 @@ the file LICENSE.md that was distributed with this source code.
 	<link rel="icon" type="image/png" sizes="32x32" href="{='resources/favicons/favicon-32x32.png'|staticFile}">
 	<link rel="icon" type="image/png" sizes="16x16" href="{='resources/favicons/favicon-16x16.png'|staticFile}">
 	<link rel="shortcut icon" type="image/x-icon" href="{='resources/favicons/favicon.ico'|staticFile}"/>
-	<link rel="manifest" href="{='resources/favicons/manifest.json'|staticFile}">
+	<link rel="manifest" href="{='resources/favicons/manifest.json'}">
```
Comment (othercorey) [inline templates/cakephp/@layout.latte:27]: Why is this change necessary?

Comment (Arhell) [reply templates/cakephp/@layout.latte:27]: Need to make the path without extra characters
https://realfavicongenerator.net/favicon_checker?protocol=https&site=api.cakephp.org%2F2.6%2F#.Yi01pXrP23A

Screenshot_1
